go/ephemeral/lib.go

```diff
@@ -179,15 +180,27 @@ func (e *EKLib) newDeviceEKNeeded(ctx context.Context, merkleRoot libkb.MerkleRo
 	s := e.G().GetDeviceEKStorage()
 	maxGeneration, err := s.MaxGeneration(ctx)
 	if err != nil {
-		return needed, err
+		switch err.(type) {
+		case *erasablekv.UnboxError:
+			e.G().Log.Debug(err.Error())
```

Comment:
Can we annotate these error messages more?

Reply:
i'll add to here: https://github.com/keybase/client/pull/12502/files#diff-97b6f490cc3dd8c39619127caf5a83a6R29

Reply:
I think it's worth adding some info to the specific call site too.

Reply:
done!